util/deploy.sh

```diff
@@ -18,6 +18,11 @@ while getopts ':ph' flag; do
   esac
 done
 
+# Ensure dependencies are installed.
+info "Installing dependencies..."
+cd ${WPTD_PATH}; make go_deps;
```

Comment:
Is there a chance that `cd` will have unwanted side effects for the rest of the script? Should we have `pushd ${WPTD_PATH} && make go_deps && popd`?

Reply:
No, I don't think it will: the next set of tasks are git-centric information gathering and then finally `gcloud app deploy --no-promote --version=w3c-move-deploy-to-utils /absolute/path/to/wptdashboard/webapp`.